courses-app/src/environments/api.config.ts

```diff
@@ -0,0 +1,14 @@
+const baseUrl = "http://localhost:3000";
+
+export const apiConfig = {
+    baseUrl: baseUrl,
+    getAuthorsUrl: baseUrl + "/authors/all",
```

Comment:
imho, this is not the best way to set env config on FE. there is a way to proxy to remote env with angular-cli. Also apiConfig in this case can be just - `getAuthorsUrl: "/authors/all",`

Reply:
fixed in the next module